internal/features/features.go

```diff
@@ -6,13 +6,15 @@ import (
 )
 
 const (
-// Add new feature gates constants (strings)
-// Ex: SomeFeature featuregate.Feature = "SomeFeature"
+	// Add new feature gates constants (strings)
+	// Ex: SomeFeature featuregate.Feature = "SomeFeature"
+	PreflightPermissions featuregate.Feature = "PreflightPermissions"
 )
 
 var operatorControllerFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	// Add new feature gate definitions
 	// Ex: SomeFeature: {...}
+	PreflightPermissions: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
It is missing `LockToDefault: true`

Reply:
making it PreAlpha to settle all (most?) concerns

Reply:
and back to Alpha b/c PreAlpha does not allow for toggling.